deepchem/__init__.py

```diff
@@ -3,7 +3,7 @@
 """
 
 # If you push the tag, please remove `.dev`
-__version__ = '2.7.2.dev'
+__version__ = '2.8.0'
```

Comment:
@shreyasvinaya We will need to readd dev tag after merge when we restart dev work to turn on pypi prerelease build